### ISSUE TITLE
Priority 1 accessibility issue: Archives works page headings not marked up

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6558]
**What**

The text ‘Career’ introduces and contextualises content. This is styled in bold text on a separate line so that this purpose is visually clear. However, it is not marked-up as a heading, so screen reader users will not be presented the purpose of this information as introductory.

<img width="523" alt="Screenshot 2021-05-27 at 16 11 30" src="https://user-images.githubusercontent.com/7315126/119851300-341da880-bf06-11eb-9f16-c01192081c4d.png">

Solution:
Please ensure semantic mark-up is used where appropriate so that the purpose of information is clear. Screen reading software relies on semantic mark-up to correctly determine information and relationships within the page to be able to present this full context to users. Headings provide context to the information they introduce and indicate page regions. Screen reader users are also able to use heading shortcuts to quickly get an idea of the contents of the page and to navigate directly to the regions on the page that they require. For more information, please refer to F2: Failure of Success Criterion 1.3.1 due to using changes in text presentation to convey information without using the appropriate markup or text.

**Where**

URL: https://wellcomecollection.org/works/hz43r7re
Page title: Francis Crick (1916-2004): archives | Wellcome Collection

**Comment by pollecuttn** [comment]
Next step: Gareth to talk to Toni Hardy

**Comment by jennpb** [comment]
My understanding of this issue: Semantic markup may not be possible given existing tools that archivists/cataloguers use. At the moment we parse the html tags but have no way of creating the semantic structure on the front-end. 

Did anyone ever talk to Toni about this issue, or investigate possible solutions?

**Comment by gestchild** [comment]
I didn't talk to Toni about this

**Comment by jennpb** [comment]
Not possible to create semantic headings based on the source data. This will need to be caveated in an accessibility statement for the site.

No further action.
